packages/expo-modules-core/ios/Core/Views/SwiftUI/SwiftUIViewHost.swift

```diff
@@ -17,4 +18,19 @@ extension ExpoSwiftUI {
       // Nothing to do here
     }
   }
+  #endif
+
+  #if os(macOS)
+  struct NSViewHost: NSViewRepresentable {
```

Comment:
Maybe it makes more sense to:
- Make `UIViewRepresentable` an alias to `NSViewRepresentable` in `Platform.swift`
- Have just one struct (`UIViewHost`) with `if` directives inside to provide platform-specific implementation

Reply:
Yeah that makes a lot of sense, thanks for the pointers <3